OOVPADatabase/XOnline/4831.inl

```diff
@@ -49,3 +49,205 @@ OOVPA_XREF(CXo_XOnlineLogon, 4831, 15,
+
+        // mov eax, ...
+        OV_MATCH(0x04, 0xb8),
+        // jmp ...
+        OV_MATCH(0x09, 0xeb),
+        // push esi
+        // mov esi, dword ptr [esp + param_2]
+        // test esi, esi
+        // push 54h
+        OV_MATCH(0x0B, 0x56, 0x8b, 0x74, 0x24, 0x0c, 0x85, 0xf6, 0x6a, 0x54),
+OOVPA_END;
+
+// ******************************************************************
+// * XOnlineMatchSearchResultsLen
+// ******************************************************************
+OOVPA_XREF(XOnlineMatchSearchResultsLen, 4831, 1+3,
+
+    XRefNoSaveIndex,
+    XRefOne)
+
+        XREF_ENTRY(0x07, XREF_XOnlineMatchSearchResultsLen_Body),
+
+        // mov ecx, ...
+        OV_MATCH(0x00, 0x8B, 0x0D),
+
+        // jmp ...
+        OV_MATCH(0x06, 0xE9),
+OOVPA_END;
+
+// ******************************************************************
+// * CXo::XOnlineMatchSearchGetResults
+// ******************************************************************
+OOVPA_XREF(CXo_XOnlineMatchSearchGetResults, 4831, 15,
+
+    XREF_CXo_XOnlineMatchSearchGetResults,
+    XRefZero)
+
+        // push ebp
+        // mov ebp, esp
+        // push ecx
+        // xor ebx, ebx
+        // cmp ecx, ebx
+        // jmp eip + $07h
+        // mov eax, ...
+        OV_MATCH(0x00, 0x55, 0x8B, 0xEC, 0x51, 0x33, 0xC0, 0x3B, 0xC8, 0x75, 0x07, 0xB8),
+
+        // mov ecx, dword ptr [ebp + param_1]
+        // push ebx
+        OV_MATCH(0x11, 0x8B, 0x4D, 0x08, 0x53),
+OOVPA_END;
+
+// ******************************************************************
+// * XOnlineMatchSearchGetResults
+// ******************************************************************
+OOVPA_XREF(XOnlineMatchSearchGetResults, 4831, 1+3,
+
+    XRefNoSaveIndex,
+    XRefOne)
+
+        XREF_ENTRY(0x07, XREF_CXo_XOnlineMatchSearchGetResults),
+
+        // mov ecx, ...
+        OV_MATCH(0x00, 0x8B, 0x0D),
+
+        // jmp ...
+        OV_MATCH(0x06, 0xE9),
+OOVPA_END;
+
+// ******************************************************************
+// * CXo::XOnlineMatchSessionUpdate
+// ******************************************************************
+OOVPA_XREF(CXo_XOnlineMatchSessionUpdate, 4831, 22,
+
+    XREF_CXo_XOnlineMatchSessionUpdate,
+    XRefZero)
+
+        // push ebp
+        // mov ebp, esp
+        // test ecx, ecx
+        // jnz eip + $07h
+        OV_MATCH(0x00, 0x55, 0x8B, 0xEC, 0x85, 0xC9, 0x75, 0x07),
+
+        // push dword ptr [ebp + param_10]
+        // push dword ptr [ebp + param_9]
+        // push dword ptr [ebp + param_8]
+        // push dword ptr [ebp + param_7]
+        // push dword ptr [ebp + param_6]
+        OV_MATCH(0x0E, 0xFF, 0x75, 0x2C, 0xFF, 0x75, 0x28, 0xFF, 0x75, 0x24, 0xFF, 0x75, 0x20, 0xFF, 0x75, 0x1C),
+OOVPA_END;
+
+// ******************************************************************
+// * XOnlineMatchSessionUpdate
+// ******************************************************************
+OOVPA_XREF(XOnlineMatchSessionUpdate, 4831, 1+23,
```

Comment:
Way too many OVs in the signature. Please trim it down to half or few. Since there is xref dependent to secure the scan's finding.

Reply:
Done.